plugins/woocommerce-admin/client/homescreen/activity-panel/orders/utils.js

```diff
@@ -74,16 +70,16 @@ export function getLowStockCount( select ) {
 	// depend on `getItemsTotalCount` to have been called.
 	// eslint-disable-next-line @wordpress/no-unused-vars-before-return
 	const totalLowStockProducts = getItemsTotalCount(
-		'products/low-in-stock',
+		'products/count-low-in-stock',
 		getLowStockCountQuery,
 		defaultValue
 	);
 
 	const isError = Boolean(
-		getItemsError( 'products/low-in-stock', getLowStockCountQuery )
+		getItemsError( 'products/count-low-in-stock', getLowStockCountQuery )
 	);
 	const isRequesting = isResolving( 'getItemsTotalCount', [
-		'products/low-in-stock',
+		'products/ount-clow-in-stock',
```

Comment:
Slight spelling/typing mistake

Reply:
Thank you for catching it! "slight" is an understatement 😄 

Fixed in https://github.com/woocommerce/woocommerce/pull/40289/commits/c487220750528ff0c30625cb9170bece82c049c7